LICENSE

```diff
@@ -1,13 +1,361 @@
-Copyright 2015 Boston Children's Hospital
+Copyright 2020 Boston Children's Hospital
```

Comment:
Feedback from Ken:

Change first line of the license file to read: © Copyright 2020 Computational Health Informatics Program, Boston Children’s Hospital, Boston, MA.

Reply:
Updated, thanks!